src/snowflake/snowpark/functions.py

```diff
@@ -8536,3 +8536,23 @@ def locate(expr1: str, expr2: ColumnOrName, start_pos: int = 1) -> Column:
     _substr = lit(expr1)
     _str = _to_col_if_str(expr2, "locate")
     return builtin("charindex")(_substr, _str, lit(start_pos))
+
+
+def enable_span_record() -> None:
+    """
+    This function enable open telemetry span recording in action functions. If opentelemetry libraries exist in the environment, span will be recorded.
```

Comment:
```suggestion
    This function enables Open Telemetry instrumentation. If Open Telemetry libraries exist in the environment, spans will be recorded.
    
    The following APIs create spans:
    - DataFrame.count()
    - DataFrame.to_pandas()
    - DataFrame.collect()
    - DataFrameWriter.copy_into_location()
    - DataFrameWriter.save_as_table()
    - UDFRegistration.register_from_file()
    - UDFRegistration.register()
    - StoredProcedureRegistration.register()
    - StoredProcedureRegistration.register_from_file()
```

We need a place to document which Snowpark APIs emit spans, can we use this method? Seems appropriate.

Reply:
@sfc-gh-yuwang  fyi